Makefile

```diff
@@ -160,6 +160,7 @@ ALL = \
 	$(ISO_ROOT)/images/$(NGINX_IMAGE_NAME)-$(NGINX_IMAGE_VERSION).tar.gz \
 	$(ISO_ROOT)/images/registry-$(REGISTRY_IMAGE_TAG).tar \
 	$(ISO_ROOT)/images/$(SALT_MASTER_IMAGE_NAME)-$(SALT_MASTER_IMAGE_VERSION).tar.gz \
+	$(ISO_ROOT)/images/$(PLATFORM_UI_IMAGE_NAME)-$(PLATFORM_UI_IMAGE_VERSION).tar.gz \
```

Comment:
Can we call this `MetalK8s UI` instead, so `$(METALK8S_UI_...)` etc?

Reply:
Yes. @MonPote ?

Reply:
Yes we can.